SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -352,5 +372,70 @@ private Task SendErrorAttachmentsAsync(Guid errorId, IEnumerable<ErrorAttachment
             }
             return Task.WhenAll(tasks);
         }
+
+        private ErrorReport BuildErrorReport(ManagedErrorLog log)
+        {
+            if (_errorReportCache.ContainsKey(log.Id))
+            {
+                return _errorReportCache[log.Id];
+            }
+            var file = ErrorLogHelper.GetStoredExceptionFile(log.Id);
+            if (file == null)
+            {
+                return null;
+            }
+            var exception = ErrorLogHelper.ReadExceptionFile(file);
+            var report = new ErrorReport(log, exception);
+            _errorReportCache.Add(log.Id, report);
+            return report;
+        }
+
+        private void ChannelSendingLog(object sender, SendingLogEventArgs e)
+        {
+            var report = ProcessEventHandlers(e, false);
```

Comment:
We don't have a convention to label parameters while invoking them, but it might be helpful for readability here